src/components/templates/Poster/Poster.tsx

```diff
@@ -0,0 +1,169 @@
+    setVenueLiveStatus(venue.id, true);
+  };
+
+  const setVenueLiveOff = () => {
+    setVenueLiveStatus(venue.id, false);
+  };
+
+  const videoParticipants = activeParticipants.map((participant) => (
+    <VideoParticipant
+      participant={participant}
+      key={participant.identity}
+      additionalClassNames="poster__video-participant"
+    />
+  ));
+
+  const hasFreeSpace = videoParticipants.length < POSTER_CELL_COUNT_MAX;
+
+  return (
+    <>
+      <div className="poster">
+        <div className="poster__header">
+          <div />
+
+          <div className="poster__header--middle-cell">
+            <p className="poster__title">{venue.title}</p>
+            <div className="poster__categories" />
+          </div>
+
+          <div className="poster__header--right-cell">
+            {isMeActiveParticipant && (
+              <div
+                className="poster__control poster__control--stop"
+                onClick={turnVideoOff}
+              >
+                <FontAwesomeIcon icon={faStop} size="lg" />
+                <span className="poster__control-text">Stop video</span>
+              </div>
+            )}
+
+            <OverlayTrigger
+              trigger="click"
+              placement="bottom-end"
+              overlay={
+                <Popover
+                  id="popover-basic"
+                  className="poster__settings-popover"
+                >
+                  <Popover.Content className="poster__settings-popover-content">
+                    <Toggler
+                      type="checkbox"
+                      className="switch-hidden-input"
+                      checked={!!isPosterLive}
+                      onChange={isPosterLive ? setVenueLiveOff : setVenueLiveOn}
+                      title={
+                        isPosterLive ? "Poster is live" : "Make poster live"
+                      }
+                    />
+                  </Popover.Content>
+                </Popover>
+              }
+              rootClose={true}
+            >
+              <div className="poster__control">
+                <FontAwesomeIcon icon={faCog} size="lg" />
+                <span className="poster__control-text">Settings</span>
+              </div>
+            </OverlayTrigger>
+
+            <div className="poster__control">
+              <FontAwesomeIcon icon={faShare} size="lg" />
+              <span className="poster__control-text">Share</span>
+            </div>
+
+            {venue.introVideoUrl && (
+              <div className="poster__control" onClick={showIntroVideoModal}>
+                <FontAwesomeIcon icon={faTv} size="lg" />
+                <span className="poster__control-text">Intro Video</span>
+              </div>
+            )}
+          </div>
+        </div>
+
+        <div className="poster__content">
+          <iframe
+            className="poster__iframe"
+            src={venue.iframeUrl}
+            title="poster-iframe"
+          />
```

Comment:
Do we want/need to set `IFRAME_ALLOW`, `allowFullscreen`, etc here? We should probably make the  `title` include something related to the poster itself if we have those details in scope (eg. poster title/similar?), not just a static string.

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1299/commits/c23cde4297f7a38d803e0c4f837e99a9c9bdccdd